tests/cli/test_cmds_spark_run.py

```diff
@@ -433,15 +434,18 @@ def test_get_docker_cmd_mrjob():
 def test_load_aws_credentials_from_yaml(tmpdir):
```

Comment:
maybe have another test to make sure missing region in yaml won't break everything.

Reply:
ack !